service/src/main/kotlin/fi/espoo/evaka/shared/auth/Acl.kt

```diff
@@ -53,7 +53,18 @@ WHERE employee_id = :userId AND daycare_id = :daycareId
     )
 
     fun getRolesForApplication(user: AuthenticatedUser, applicationId: UUID): AclAppliedRoles = AclAppliedRoles(
-        (user.roles - UserRole.ACL_ROLES) + Database(jdbi).read {
+        (user.roles - UserRole.ACL_ROLES) + Database(jdbi).read { it ->
+            val assistanceNeeded = it.createQuery(
+                // language=SQL
+                """
+SELECT (f.document -> 'careDetails' ->> 'assistanceNeeded')
+FROM application a inner join application_form f on a.id = f.application_id and f.latest = true
```

Comment:
Since we already have an application ID, maybe the `application` table can be skipped?

```
...
FROM application_form f
WHERE f.application_id = :applicationId AND f.latest IS TRUE
```